scripts/darwin.sh

```diff
@@ -62,5 +66,23 @@ for disk in $DISKS; do
     echo "system: False"
   fi
 
+  partitions="`diskutil list $disk | get_device_identifiers`"
+  partitionmountpoints=""
+
+  for partition in $partitions; do 
+    partitioninfo="`diskutil info $partition`"
```

Comment:
BTW, there are some very cool suggestions on the original issue about how to list available drives, including partitions. In particular, there is one using just `mount`, which means we can cache `mount`'s output at the very beginning and use it for everything.